app/views/partials/_navbar.html.erb

```diff
@@ -0,0 +1,18 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light" id="navigationBar" >
+    <a class="navbar-brand abs" href="#">Kitchen to Couch</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsingNavbar">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="navbar-collapse collapse" id="collapsingNavbar">
+        <ul class="navbar-nav">
+            <li class="nav-item active">
+                <%= link_to 'Home', root_path, class: 'nav-link' %>
+            </li>
+        </ul>
+        <ul class="navbar-nav ml-auto">
+            <li class="nav-item">
+                <a class="nav-link" href="" data-target="#myModal" data-toggle="modal">Cart</a>
```

Comment:
How about that modal link, what does it do?

Reply:
Same here with the standard bootstrap navbar template, but I understand what you mean and we'll take it out.

Reply:
removed the modal, we'll use the link later on when we have the order page